7-ns.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title: Namespace for RDF
+permalink: /ns
+nav_order: 7
+---
+## PaymentPointer
+
+Use `http://paymentpointers.org/ns#PaymentPointer` as the predicate when linking a WebID to a Payment Pointer. Make sure this triple lives in the profile document to which the WebID dereferences. Example:
```

Comment:
I didn't look at this closely enough in my first review.

Following the URL conventions for ontologies, I thought `http://paymentpointers.org/ns#PaymentPointer` was a class (due to CamelCase).
However, it looks like this is here used as a predicate, for which `http://paymentpointers.org/ns#paymentPointer` (mixedCase) would be more appropriate.

I would however suggest a more generic solution, so we can leave the door open for other types of payment pointers in the future.
For example:

```turtle
<#me> a <foaf:Person> ;
  pp:hasPaymentPointer [
    a pp:InterledgerPaymentPointer;
    pp:paymentPointerValue "$alice.wallet.example".
  ].
```

Such a reified form would also allow people to include multiple pointers, and annotate them for different purposes. For example, one wallet could be used for work-related matters, while another could be used for personal things.